### PR TITLE
Team pages

### LINKED ISSUE #6: Team Pages
This would include:
- Roster (with a few stats & links to player pages)
- Team stats (I wouldn't consider myself even an amateur statistician, but one of us can figure something out)
- Standings place
- Last five/ten games (last five, next five would be good if the nhl gives scheduled games)

### PR DESCRIPTION
**Pull request by bsummy**
Fixes #6 

Buckle Up - this is a big one. I hope that you'll take a look over everything when you get some time, seeing as this is my first larger commit. It also took care of a couple smaller issues that I noticed around the app, but I'll try to note them here:

- New error page since bad team pages also route there (I changed the error page photo to spider gritty because I thought the picture goes hard)
- new NHL api gives height in inches, not feet and inches. 2024 players were coming through in the wrong form.
- standings now keep track of their own timestamps and link to team pages on the team icon

There's two things that I want to include before this is merged
1. need to make sure I'm not overwriting the .db files, so after you approve, I need you to take the SQL statements (in backend/sql) and run them in the most up to date version of player_stats.db. I'll remove any changes I made to the db files from the commit.
2. if possible, can I get the default current timestamp added to the team stats db? if you look at the standings page you'll see what I plan to do.

Something I've been meaning to ask:
- Do you have it setup to write new sets of standings to the same table and not drop/recreate it each week? With the new lastUpdated stuff it should be setup to only take the most recent set of standings, but I haven't tested it yet since I haven't had more than one week of standings.

### CODE REVIEW

**Comment by neilpl24** [comment]
> Do you have it setup to write new sets of standings to the same table and not drop/recreate it each week? With the new lastUpdated stuff it should be setup to only take the most recent set of standings, but I haven't tested it yet since I haven't had more than one week of standings. 

Yes! Would be appending instead of replacing in the pandas to sql function

File: backend/dev.js [hunk 1]
```diff
@@ -294,8 +311,16 @@ app.get("/players/info/:id", (req, res, next) => {
   }
 
   db.get(
-    `SELECT * FROM players${season}
-            WHERE player_id = ?`,
+    `WITH team_ids AS(
+        SELECT team_id,
+        CASE WHEN team_name = 'Montreal Canadiens' THEN 'Montréal Canadiens' ELSE team_name
```
Comment (neilpl24) [inline backend/dev.js:316]: Gonna put in a different PR to eventually fix this by just defaulting the scraper/everything else to the english spelling

Comment (bsummy) [reply backend/dev.js:316]: yeah I saw your comments in the code and I figured I didn't need the permanent fix :)

**Comment by bsummy** [comment]
I resolved the merge issues. If you're reviewing and the app doesn't look right, just run the create & insert in backend/sql in player_stats.db and restart the connection and it should work

**Review by neilpl24 — CHANGES_REQUESTED** [review]
Nice work! Few comments I have on the design:

- Team links/player links should either be hyperlinked or some sort of hover CSS property to indicate that is supposed to be clicked on.
- I think for the team statistics, you could add some visual component with color gradients or some other type of viz. Being vague here on purpose to let you explore that
- Team page looks good, I do think for the roster section, a table would be a better fit, could thrown in headshots as well

Good stuff, inspired some other ideas of mine

**Pull request merged**